scripts/run_experiment.py

```diff
@@ -2,9 +2,9 @@
 import ast
```

Comment:
```
(garage) garage (fix_sleeping_proc *) $ python scripts/run_experiment.py
2018-08-17 09:45:25.883375 PDT | tensorboard data will be logged into:/Users/jonathon/Documents/garage/garage/data/experiment_2018_08_17_09_45_25_880884_PDT_95e7c
Traceback (most recent call last):
  File "scripts/run_experiment.py", line 201, in <module>
    run_experiment(sys.argv)
  File "scripts/run_experiment.py", line 187, in run_experiment
    data = pickle.loads(base64.b64decode(args.args_data))
  File "/anaconda2/envs/garage/lib/python3.6/base64.py", line 80, in b64decode
    s = _bytes_from_decode_data(s)
  File "/anaconda2/envs/garage/lib/python3.6/base64.py", line 46, in _bytes_from_decode_data
    "string, not %r" % s.__class__.__name__) from None
TypeError: argument should be a bytes-like object or ASCII string, not 'NoneType'
```

Reply:
You forgot an argument. `run_experiment.py` is usually called by `garage.misc.instrument.run_experiment`